app/src/main/res/values/strings.xml

```diff
@@ -339,6 +339,10 @@
     <string name="livestreams">Livestreams</string>
     <string name="nsfw">NSFW</string>
     <string name="others">Others</string>
+    <plurals name="episodes">
```

Comment:
<string name="episode">Episode</string>
    <string name="episodes">Episodes</string>

already exists, this will just make extra translation

Reply:
Ah I was aware it did but using getQuantityString made for cleaner code in the adapter. I can change it back though if you want?

Reply:
Even if it is cleaner, it would be much better to do all plurals in a seperate PR and on all languages. Just doing it on english on a single item will be a real PITA to maintain and fix as you now have 2 translation keys on the same thing. 

Also before you do that PR, check that the translation site can handle plurals.

Reply:
@fire-light42 what if I do:
```xml
<plurals name="episodes" translatable="false">
    <item quantity="one">@string/episode</item>
    <item quantity="other">@string/episodes</item>
</plurals>
 ```
 
 ? If not I can just revert that part no problem but if you are fine with this method I'd personally prefer it.

Reply:
I did this but will revert if you don't like it this way.